v3.0/releases/3.0.1.md

```diff
@@ -13,6 +13,7 @@ TiDB Ansible version: 3.0.1
 
 ## TiDB
 
++ Add support for the `MAX_EXECUTION_TIME` feature [#11026](https://github.com/pingcap/tidb/pull/11026)
```

Comment:
We also have this file in dev and v2.1 currently. Would you please also update them?

Reply:
It's already in the 2.1.14 release note: 
https://github.com/pingcap/docs/blob/master/v2.1/releases/2.1.14.md
@lilin90